### DISCUSSION TITLE
Bugged Contributeurs button.

### DISCUSSION THREAD

**Discussion by nxtlo** [community/community#12168] [Mobile]
When clicking on the contributors button it instantly quits the application. Has this been reported before?

**Comment by stevepopovich** [comment]
Hi @nxtlo can you add what platform you are seeing this on and specific steps to repro this bug?

**Reply by nxtlo** [reply]
Ofcourse.

Platform
- iPHONE Xr IOS 15.3.1
- Github Version: 1.53.0

Steps to reproduce:
- Click on Github application.
- Go to any repo.
- Click at the contributors button to view people who contributed to the repo.
- Application quits.

Hopefully this is helpful.

**Reply by nxtlo** [reply]
Live action.

https://user-images.githubusercontent.com/35225215/156018863-bc39c59a-89e9-4323-b526-b37d6fbb8b81.MOV

**Reply by stevepopovich** [reply]
Perfect. Passing along to the team! Thanks.
